<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netgen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSaveMesh"/>
    <addaction name="separator"/>
    <addaction name="actionExportMesh"/>
    <addaction name="separator"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Geometry">
    <property name="title">
     <string>&amp;Geometry</string>
    </property>
    <addaction name="actionScanCSGGeometry"/>
    <addaction name="actionCSGOption"/>
   </widget>
   <widget class="QMenu" name="menu_Mesh">
    <property name="title">
     <string>&amp;Mesh</string>
    </property>
    <addaction name="actionStartMeshing"/>
    <addaction name="actionStopMeshing"/>
    <addaction name="separator"/>
    <addaction name="actionRefineUniform"/>
    <addaction name="separator"/>
    <addaction name="actionMeshingOptions"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewGeometry"/>
    <addaction name="actionViewMesh"/>
    <addaction name="separator"/>
    <addaction name="actionViewTerminal"/>
    <addaction name="separator"/>
    <addaction name="actionImageSetup"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Geometry"/>
   <addaction name="menu_Mesh"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveMesh"/>
   <addaction name="separator"/>
   <addaction name="actionViewTerminal"/>
   <addaction name="separator"/>
   <addaction name="actionStartMeshing"/>
   <addaction name="actionStopMeshing"/>
   <addaction name="separator"/>
   <addaction name="actionMeshingOptions"/>
   <addaction name="separator"/>
   <addaction name="actionImageSetup"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="statusTip">
    <string>Open Geometry/Mesh files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L, Ctrl+G</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="actionSaveMesh">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="statusTip">
    <string>Save Mesh file</string>
   </property>
  </action>
  <action name="actionScanCSGGeometry">
   <property name="text">
    <string>&amp;Scan CSG Geometry</string>
   </property>
  </action>
  <action name="actionCSGOption">
   <property name="text">
    <string>CSG &amp;Options...</string>
   </property>
  </action>
  <action name="actionCSGProperties">
   <property name="text">
    <string>CSG &amp;Properties...</string>
   </property>
  </action>
  <action name="actionStartMeshing">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Generate Mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G, Ctrl+M</string>
   </property>
  </action>
  <action name="actionStopMeshing">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop Meshing</string>
   </property>
  </action>
  <action name="actionMeshingOptions">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/setup.png</normaloff>:/images/setup.png</iconset>
   </property>
   <property name="text">
    <string>Meshing &amp;Options...</string>
   </property>
  </action>
  <action name="actionLoadMesh">
   <property name="text">
    <string>Load &amp;Mesh...</string>
   </property>
  </action>
  <action name="actionExportMesh">
   <property name="text">
    <string>&amp;Export Mesh...</string>
   </property>
   <property name="statusTip">
    <string>Export Mesh file to different formats</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="statusTip">
    <string>Close current model</string>
   </property>
  </action>
  <action name="actionViewGeometry">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Geometry</string>
   </property>
   <property name="statusTip">
    <string>Show Geometry</string>
   </property>
  </action>
  <action name="actionViewMesh">
   <property name="text">
    <string>&amp;Mesh</string>
   </property>
   <property name="statusTip">
    <string>Show Mesh</string>
   </property>
  </action>
  <action name="actionViewTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/terminal.png</normaloff>:/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Terminal</string>
   </property>
   <property name="statusTip">
    <string>Show Terminal</string>
   </property>
  </action>
  <action name="actionRefineUniform">
   <property name="text">
    <string>&amp;Refine</string>
   </property>
   <property name="statusTip">
    <string>Refine uniform</string>
   </property>
  </action>
  <action name="actionRefineSecondOrder">
   <property name="text">
    <string>&amp;Second order</string>
   </property>
   <property name="toolTip">
    <string>Second order</string>
   </property>
   <property name="statusTip">
    <string>Refine second order</string>
   </property>
  </action>
  <action name="actionRefineHighOrder">
   <property name="text">
    <string>&amp;High order</string>
   </property>
   <property name="statusTip">
    <string>Refine high order</string>
   </property>
  </action>
  <action name="actionRefineOptions">
   <property name="text">
    <string>Refine &amp;Options...</string>
   </property>
   <property name="statusTip">
    <string>Setup refine parameters</string>
   </property>
  </action>
  <action name="actionRefineBisection">
   <property name="text">
    <string>&amp;Bisection</string>
   </property>
  </action>
  <action name="actionImageSetup">
   <property name="icon">
    <iconset resource="Netgen.qrc">
     <normaloff>:/images/parameters.png</normaloff>:/images/parameters.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Setup...</string>
   </property>
   <property name="statusTip">
    <string>Setting up model image parameters</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Netgen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStartMeshing</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotGenerateMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStopMeshing</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotStopMeshing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCSGOption</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotCSGOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMeshingOptions</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMeshingOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveMesh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScanCSGGeometry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotScanCSG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewGeometry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotViewGeometry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewMesh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotViewMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewTerminal</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotViewTerminal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportMesh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotExportMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefineUniform</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotRefineUniform()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImageSetup</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotImageSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOpen()</slot>
  <slot>slotGenerateMesh()</slot>
  <slot>slotStopMeshing()</slot>
  <slot>slotCSGOptions()</slot>
  <slot>slotMeshingOptions()</slot>
  <slot>slotSave()</slot>
  <slot>slotScanCSG()</slot>
  <slot>slotClose()</slot>
  <slot>slotViewGeometry()</slot>
  <slot>slotViewMesh()</slot>
  <slot>slotViewTerminal()</slot>
  <slot>slotExportMesh()</slot>
  <slot>slotRefineUniform()</slot>
  <slot>slotRefineSecondOrder()</slot>
  <slot>slotRefineHighOrder()</slot>
  <slot>slotRefineBisection()</slot>
  <slot>slotImageSetup()</slot>
 </slots>
</ui>
